src/screens/ErrorHandler/state/reducer.test.ts

```diff
@@ -1,71 +0,0 @@
-import each from 'jest-each';
```

Comment:
Jeg er sikker på at det er en god grunn, men hvorfor fjerner dere denne testen?

Reply:
de testene ble ikke lenger gjeldene siden håndtering av logikk er flyttet fra actions til reducers for errorhandleren, så fjernet dem. Men er vel kanskje noe av den nye logikken som kan/bør testes. Skal se på det.

Reply:
Det er kanskje @stromland som bør bestemme det siden det ikke er en klar teststrategi for konsollet. Det er vel litt hipp som happ hva som faktisk testes.